### ISSUE TITLE
Potential bug: depth proximity penalty is never applied because terminations is boolean

### ISSUE THREAD

**Issue by VaporTang** [ntnu-arl/aerial_gym_simulator#64]
Hi, thanks for maintaining Aerial Gym!

While inspecting the navigation task at commit `f0d0f05`, I noticed a possible issue with the depth proximity penalty.

### Relevant code

In `aerial_gym/env_manager/env_manager.py`, `crashes` is initialized as a boolean tensor:

```python
self.global_tensor_dict["crashes"] = torch.zeros(
    (self.num_envs),
    device=self.device,
    requires_grad=False,
    dtype=torch.bool,
)
```

In `aerial_gym/task/navigation_task/navigation_task.py`, `self.terminations` references this tensor. However, `post_image_reward_addition()` uses:

```python
self.rewards[self.terminations < 0] += -exponential_reward_function(
    4.0, 1.0, self.min_pixel_dist[self.terminations < 0]
)
```

### Observed behavior

For a boolean tensor, `self.terminations < 0` is always false:

```python
import torch

terminations = torch.tensor([False, True])
print(terminations < 0)
# tensor([False, False])
```

I also executed the original `post_image_reward_addition()` method in an isolated CPU test with synthetic depth observations. The depth penalty did not change the rewards.

### Expected behavior / question

If this penalty is intended to apply to non-colliding environments, should the mask be `~self.terminations` or `self.terminations == 0`?

Or is disabling this penalty intentional?

Thank you!
